projects/pffft/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+WORK_DIR=$WORK/pffft
+SRC_DIR=$SRC/pffft
+cd $WORK_DIR
+
+# Building PFFFT as a static library.
+if [ -f libpffft.a ]; then
+  rm libpffft.a
+fi
+gcc -c -msse2 -O3 -Wall -W -lm -fpic $SRC_DIR/pffft.c -o pffft.o
```

Comment:
why gcc? 
If you don't build with the provided CXX and CXXFLAGS (CC/CFLAGS) you won't get much from fuzzing the code
(no coverage feedback, no sanitizers)

Reply:
Thanks! Fixed.